src/3rdparty/mjcpy2/mjcpy2_getmodel_autogen.i

```diff
@@ -156,7 +156,7 @@
     out["tex_rgb"] = toNdarray2<mjtByte>(m_model->tex_rgb, m_model->ntexdata, 1);
     out["mat_texid"] = toNdarray2<int>(m_model->mat_texid, m_model->nmat, 1);
     out["mat_texuniform"] = toNdarray2<mjtByte>(m_model->mat_texuniform, m_model->nmat, 1);
-    out["mat_texscale"] = toNdarray2<float>(m_model->mat_texscale, m_model->nmat, 2);
+    out["mat_texrepeat"] = toNdarray2<float>(m_model->mat_texrepeat, m_model->nmat, 2);
```

Comment:
I guess mat_texrepeat is not used somewhere? I would expect it to be used in the renderer, but I guess not. Could probably just remove.